app/pages/reporter/complete-submit.tsx

```diff
@@ -16,43 +19,94 @@ class CompleteSubmit extends Component<Props> {
   static allowedGroups = ALLOWED_GROUPS;
   static isAccessProtected = true;
   static query = graphql`
-    query completeSubmitQuery {
+    query completeSubmitQuery($applicationId: ID!) {
       query {
         session {
           ...defaultLayout_session
         }
+        application(id: $applicationId) {
+          facilityByFacilityId {
+            facilityName
+            organisationByOrganisationId {
+              operatorName
+            }
+          }
+          latestSubmittedRevision {
+            versionNumber
+          }
+        }
       }
     }
   `;
 
   render() {
     const {
-      query: {session}
+      query: {session, application},
+      router
     } = this.props;
+
+    const {applicationId} = router.query;
+    const {versionNumber} = application.latestSubmittedRevision;
+    const {facilityName} = application.facilityByFacilityId;
+    const {
+      operatorName
+    } = application.facilityByFacilityId.organisationByOrganisationId;
+
     return (
-      <DefaultLayout
-        session={session}
-        title={
-          <>
-            Thank you for your submission. Your application has been sent and is
-            being reviewed.
-            <br /> We will notify you as soon as there is an update.
-          </>
-        }
-      >
+      <DefaultLayout session={session}>
+        <Row className="justify-content-md-center mb-5">
+          <Col>
+            <ProgressStepIndicator
+              title="Your Facility's Application Status"
+              steps={[
+                {
+                  description: 'Complete Facility CIIP Application',
+                  badgeStyle: StatusBadgeColor.NONE,
+                  number: 1,
+                  completed: true
+                },
+                {
+                  description: 'Submit Completed Application',
+                  badgeStyle: StatusBadgeColor.INITIAL,
+                  number: 2,
+                  completed: true
+                },
+                {
+                  description: 'Application Reviewed by Administrators',
+                  badgeStyle: StatusBadgeColor.PENDING,
+                  number: 3
+                },
+                {
+                  description: 'Application Approved or Rejected',
+                  badgeStyle: StatusBadgeColor.APPROVED,
+                  number: 4
+                }
+              ]}
+            />
```

Comment:
I'd be tempted to refactor this in a `CiipApplicationProgressStepIndicator` component, to avoid duplication with `pages/reporter/facilities.tsx`

Reply:
Yeah I was wondering about that. The text isn't the same between the screenshots, so I'm not sure if I should reuse the labels of the facilities page or not.